recipes/dftatom/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "dftatom" %}
+{% set version = "1.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/certik/dftatom/archive/v{{ version }}.tar.gz
+  sha256: 7a34311cc2ca5641a2ba81ea4809c972e36ff0e1798498cf6aa9f365ead5ba59
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('fortran') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+  host:
+  run:
+
+test:
+  commands:
+    - test -f ${PREFIX}/lib/libdftatom.so  # [linux]
+    - test -f ${PREFIX}/lib/libdftatom.dylib  # [osx]
+    # - if exist %LIBRARY_PREFIX%\lib\??? (exit 0) else (exit 1)  # [win]
+
+about:
+  home: https://github.com/certik/dftatom
+  license: MIT
+  license_family: MIT
+  license_file: License
+  summary: 'Routines for Radial Integration of Dirac, Schrödinger, and Poisson Equations'
+  description: |
+    This library implements accurate and efficient radial Schrödinger and Dirac shooting-method solvers. 
+    They work with any grid and any numerical potential. In addition, it also implements a radial 
+    Poisson solver and density functional theory self-consistency cycle.
+  doc_url: https://github.com/certik/dftatom/
+
+extra:
+  recipe-maintainers:
```

Comment:
@wolfv Don't you want to be a maintainer?

Reply:
I don't actually know anything about dftatom so ... I think it's fine to just have @certik as maintainer.
Also there are no frequent releases I think :)